opstack/src/consensus.rs

```diff
@@ -136,6 +152,79 @@ impl Inner {
     }
 }
 
+fn verify_unsafe_signer(server_url: String, signer: Arc<Mutex<Address>>) {
+    #[cfg(not(target_arch = "wasm32"))]
+    let run = tokio::spawn;
+
+    #[cfg(target_arch = "wasm32")]
+    let run = wasm_bindgen_futures::spawn_local;
+
+    run(async move {
+        let eth_config = mainnet();
```

Comment:
We should at a minimum support setting a different checkpoint (the `-w` flag) and fetching the fallback (the `-l` flag) in the cli. Lets add those and make sure to document what it means (that it is for the ethereum sync) in the comments above the flag param in the cli struct (clap uses this to generate `--help`.